python/cudf/cudf/core/dataframe.py

```diff
@@ -1845,7 +1845,7 @@ def rename(self, mapper=None, columns=None, copy=True, inplace=False):
             for key, col in self._data.items():
                 if key in mapper:
                     if mapper[key] in out.columns:
-                        out_column = mapper[key] + ("cudf_" + str(postfix),)
+                        out_column = mapper[key] + "cudf_" + str(postfix)
```

Comment:
What is the Pandas behavior here? To allow duplicate column names?

Reply:
Yep, pandas allows duplicate column names.

Reply:
Got it. Would suggest we just do `mapper[key] + "_" + str(postfix)` then as `z` --> `zcudf_1` is super messy whereas `z_1` seems better.

Reply:
Done 👍